### ISSUE TITLE
Trajectory calculation

### ISSUE THREAD

**Issue by Jeslone** [proroklab/minicar#1]
Hello. Good Job. 
I'm Jeslone. I'm very interested and I'd like to replicate the experiment. But I don't see which camera and where you placed it as long as I know that you used reflective markers and a kalman filter! 

How the lane information is sent to the minicar via WiFi?

Can you please give me more details ?

**Comment by zlg9folira** [comment]
The [paper](https://arxiv.org/pdf/1902.06133.pdf) published along with this project, describes how the cars localize themselves relatively. An OptiTrack motion capture system is used along with 5 markers on a vehicle and a series of calculation to further estimate the vehicle dynamics.

**Comment by nhyldmar** [comment]
Thank you @zlg9folira. This is how we did localisation.

> How the lane information is sent to the minicar via WiFi?

We processed information on a central computer and sent the desired velocity and steering angle (and headlight brightness) to each car over a network. The packets were sent via UDP to the Raspberry Pi Zero W on each minicar over a local network. The firmware we wrote then applied these received values to the motor and servo (and LEDs).

Hope this helps. Reopen this issue if this doesn't answer your questions.
